docs/Components/BaseComponent.md

```diff
@@ -0,0 +1,12 @@
+# Base Component
+
+## Append child function
+This function is responsible for adding a child to a component
+``` python
+from adr.Components import BaseComponent
+
+base_component = BaseComponent('component', 'generic_component', 3.4)
+child_component = BaseComponent("wing1", "wing", 1.1)
+
+base_component.append_child(child_component)
+```
```

Comment:
It would be nice to print the child_component later, as you did with the gravitational_center property, so the user who reads this doc understand how to access the child component (both on the children dictionary and the attribute way).